uaclient/status.py

```diff
@@ -214,11 +214,13 @@ def format_tabular(status: "Dict[str, Any]") -> str:
     content = [STATUS_HEADER]
     for service_status in status["services"]:
         entitled = service_status["entitled"]
+        uf_status = service_status["userFacingDetails"]
```

Comment:
`uf_status` is a very confusing variable name for this. `service_status["status"]` is what we refer to as user-facing status everywhere else.

Reply:
calling it uf_details.